Document running a local version of lightning on the cloud (#15962)
Lightning-AI/lightning: .github/CONTRIBUTING.md

  git push -f
  ```

> #### How to run an app on the cloud with lightning that you're making changes

Reviewer: ```suggestion
#### How to run an app on the cloud with a local version of lightning
```

> 

Reviewer: can you add some motivation here for why you would want to use this

Author: PTAL!

Reviewer 2: @akihironitta added? :otter:

Author: Yes! I belive so :)

The lightning cloud uses the latest release by default. However, you might want to run your app with lightning that you have made changes to in your directory. To use your local version of lightning on the cloud, set the following environment variable:

```bash

> export PACKAGE_LIGHTNING=1  # <- this is the magic to use local version!

Reviewer 2: ```suggestion
export PACKAGE_LIGHTNING=1  # <- this is the magic to use your version (not mainstream PyPI)!
```
